assets/scss/bolt.scss

```diff
@@ -1,5 +1,9 @@
+@import './themes/default';
 @import './vendor/bootstrap/bootstrap';
-@import './init/init';
+@import './vendor/bootstrap-vue/bootstrap-vue';
+
+/* old */
```

Comment:
One of _the_ main advantages of Git is version control. Feel free to delete these "old" things, because if we might need them one day, we can simply go back in git history. 

Doing so keeps the code base nice and tidy. 

(in other words: Comments are for comments. Not for deleted code)